sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample8_ExtractSummary.md

```diff
@@ -0,0 +1,118 @@
+
+```C# Snippet:TextAnalyticsExtractSummaryOperationStatus
+// View operation status.
+Console.WriteLine($"AnalyzeActions operation has completed");
+Console.WriteLine();
+
+Console.WriteLine($"Created On   : {operation.CreatedOn}");
+Console.WriteLine($"Expires On   : {operation.ExpiresOn}");
+Console.WriteLine($"Id           : {operation.Id}");
+Console.WriteLine($"Status       : {operation.Status}");
+Console.WriteLine($"Last Modified: {operation.LastModified}");
+Console.WriteLine();
+```
+
+To view the final results of the long-running operation:
+
+```C# Snippet:TextAnalyticsExtractSummaryAsyncViewResults
+// View operation results.
+foreach (AnalyzeActionsResult documentsInPage in operation.GetValues())
+{
+    IReadOnlyCollection<ExtractSummaryActionResult> summaryResults = documentsInPage.ExtractSummaryResults;
+
+    foreach (ExtractSummaryActionResult summaryActionResults in summaryResults)
+    {
+        if (summaryActionResults.HasError)
+        {
+            Console.WriteLine($"  Error!");
+            Console.WriteLine($"  Action error code: {summaryActionResults.Error.ErrorCode}.");
+            Console.WriteLine($"  Message: {summaryActionResults.Error.Message}");
+            continue;
+        }
+
+        foreach (ExtractSummaryResult documentResults in summaryActionResults.DocumentsResults)
+        {
+            if (documentResults.HasError)
+            {
+                Console.WriteLine($"  Error!");
+                Console.WriteLine($"  Document error code: {documentResults.Error.ErrorCode}.");
+                Console.WriteLine($"  Message: {documentResults.Error.Message}");
+                continue;
+            }
+
+            Console.WriteLine($"  Extracted the following {documentResults.Sentences.Count} sentence(s):");
+            Console.WriteLine();
+
+            foreach (SummarySentence sentence in documentResults.Sentences)
+            {
+                Console.WriteLine($"  Sentence: {sentence.Text}");
+                Console.WriteLine($"  Rank Score: {sentence.RankScore}");
+                Console.WriteLine($"  Offset: {sentence.Offset}");
+                Console.WriteLine($"  Length: {sentence.Length}");
+                Console.WriteLine();
+            }
+        }
+    }
+}
+```
+
+To see the full example source files, see:
+
+* [Synchronously ExtractSummary](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/README.md)
+* [Asynchronously ExtractSummary](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/README.md)
+* [Synchronously ExtractSummary Convenience](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/README.md)
+* [Asynchronously ExtractSummary Convenience](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/textanalytics/Azure.AI.TextAnalytics/README.md)
```

Comment:
Can't link the correct files since they're not merged into GitHub yet. Created blocking issue https://github.com/Azure/azure-sdk-for-net/issues/23098 for tracking.